### PR TITLE
Add divisionEvents

### PR DESCRIPTION
**Pull request by amberwebb**

### CODE REVIEW

File: docs/collections/division_events.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* `callback`: [function] - callback to be executed when the operation completes.
+
+### Examples
+```javascript
+// ~~~~~
+// Loads all divisionEvents for `divisionId: 1`.
+teamsnap.loadDivisionEvents(1);
+
+// ~~~~~
+// Loads all divisionEvents for `id: 1`.
```
Comment (matthopson) [inline docs/collections/division_events.md]: Should this be "Loads divisionEvent for `id: 1`" ?

**Comment by amberwebb** [comment]
@matthopson Please review again.

**Comment by amberwebb** [comment]
@matthopson Updated version and build. Are I good to merge?

Also, since I don't need this update for njw, should this wait until the next update to go into the njw codebase?

**Comment by amberwebb** [comment]
@matthopson How should I exclude them?

**Pull request merged**